package-lock.json

```diff
@@ -114,7 +114,7 @@
         },
```

Comment:
No package.json changes, shouldn't be adding a lock file here

Reply:
Thought I had backed that out already, done!